### ISSUE TITLE
Feeding without using timer

### ISSUE THREAD

**Issue by myxor** [MrApplejuice/BabyBuddyAndroid#11]
Hello @MrApplejuice,

first of all: i like your app.

Any plans to integrate the functionality to add a feeding without the need to use timer?
Similar to the way you did it for Diaper change?

**Comment by MrApplejuice** [comment]
Hey there! Wow... this is surprising, thanks for giving the app a try already :smile: Did not expect feedback so early, given that it is not really listed anywhere.

Anyway, yes - I was planning to do this by introducing a grace time for very short durations to just collapse into 0-second entries. From the user's perspective, this would mean that double-tapping a timer would register a moment instead of a timerange. It still will still require the creation of timers in babybuddy - not sure if you want to avoid that - but it allows one to register e.g. feedings or tummy-times that babybuddy recognizes as moments as well.

Not implemented right now. In the long run, I am checking if I can manage the timers themselves in some smarter manner as well as that is the most obscure bit of the app right now.

**Comment by myxor** [comment]
I was searching for "Baby Buddy" on Github and ended up here :)

I think you misunderstood my suggestion: 
We are using the timer functionality of Baby Buddy almost to never. Most of the time when feeding we know at which time we started feeding and just add the entry to BB after the feeding using the feeding->add menu and type in the start and end time.
I would like to do this via your app as well.

**Comment by MrApplejuice** [comment]
> Most of the time when feeding we know at which time we started feeding and just add the entry to BB after the feeding using the feeding->add menu and type in the start and end time.

Yes, I recognize this usecase. Right now the only way to do this is using the timer-functions. I recorded a video on how it was designed - but I also agree that this part, while really handy, is also very wonky:

https://user-images.githubusercontent.com/542105/163202244-15f99a41-bc04-4743-9dd1-e44e33f19dd6.mp4

In the video I demonstrate how you can utilize double-clicks on the timers to do pretty much what you want. We at least are happy enough with using the timers this way. The only difference right now is that it will still register as a 1-2 second period and not as start and end time being equal. That mostly does not matter much but it pollutes the "time stats" overview for different activities, in case you use those overviews of babybuddy. And of course there remain some "stopped" timers in the babybuddy webapp.
